recipe/meta.yaml

```
@@ -6,27 +6,33 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 6fa44c97a412b4dc51e2f5fd4dc73c5bd499d35280f88cd2bcd26f8ae1e4577b
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
  skip: True  # [py<37]
```

https://github.com/zopefoundation/zope.proxy/blob/5.0.0/setup.py#L124